cmd/configschema/go.mod

```diff
@@ -544,6 +544,7 @@ require (
 	go.opentelemetry.io/otel/metric v0.31.0 // indirect
 	go.opentelemetry.io/otel/sdk v1.8.0 // indirect
 	go.opentelemetry.io/otel/trace v1.8.0 // indirect
+	go.opentelemetry.io/proto/otlp v0.18.0 // indirect
```

Comment:
Where is this coming from?

Reply:
I am not sure yet, I am surprised to see that as well.  If I run `make gotidy` it shows up.

Reply:
I'll look into it more tomorrow

Reply:
@bogdandrutu it is coming from the fact that we use the proto libraries to help build our enum symbol table: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d00fbea7b96e75d67571696deccb2909cef806b6/pkg/telemetryquerylanguage/contexts/tqltraces/traces.go#L64

Reply:
You should use `ptrace.SpanKind[Unspecified|Internal|...]`

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12905